src/site/fml/faq.fml

```diff
@@ -29,5 +29,36 @@
                 </p>
             </answer>
         </faq>
+        <faq id="How can I provide my own logger">
+            <question>How can I provide my own logger implementation?</question>
+            <answer>
+                <p>
+                    Seshat uses the java <a href="https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/ServiceLoader.html">ServiceLoader</a>
+                    concept in order to load different LoggerFactory implementation. This means that if you want to
```

Comment:
* to load different Loggerfactory implementations - plural

* This means that, if you want to - comma

Reply:
Fixed.